### PR TITLE
feat: add gallery modal empty state

### LINKED ISSUE #667: (SP: ) Add missing not found UI for images in gallery modal
## Description
As part of the [US-Admin-IMG-4 — View the Gallery with search](https://github.com/Liatoshynsky-Foundation/lf-client/issues/1051)
Currently, when the gallery modal is empty or when a search query returns no results, the gallery displays a blank state. This can be confusing for users as it lacks feedback. This task involves implementing an empty state UI that provides a clear message to the user, explaining that no images were found. This will improve the overall user experience by providing context and a clear call to action.

## Acceptance Criteria
- [ ] **Empty State Visibility:** The empty state message doesn't appear when the image array is empty or when a search filter returns zero results.
- [ ] **Localization:** The text must be implemented using the project's localization system (i.e., i18n) with the following translations:
    - **EN:** "No images found. Try adding images to the media library."
    - **UA:** "Зображень не знайдено. Спробуйте додати зображення до медіатеки."
- [ ] **Layout:** The message should be centered horizontally and vertically within the image gallery container.
- [ ] **Loading State:** The empty state should only appear after the data fetching is complete (it should not flash while `isLoading` is true).
- [ ] **Unit Tests**:
- [ ] Test that the message renders when the image array is empty.
- [ ] Test that the message does not render when images are present.
- [ ] Test that the correct translation is displayed based on the locale.

## Steps to Implement
1. Locate the component responsible for rendering the gallery modal.
2. Create or update the UI to display the "No images found" text.
3. Integrate localization for both English and Ukrainian strings.
4. Apply styling to ensure the text is properly aligned within the UI.
5. Add unit tests to cover the empty and non-empty states.

## Extra
**Current UI Issue:**
Image

### PR DESCRIPTION
**Pull request by exity2905**
## Description

Adds an empty state message to the media gallery modal when no items are available or when search/filtering returns no results.

The empty state uses Ukrainian text for the admin UI and supports image, audio, and PDF gallery modes.

Closes Liatoshynsky-Foundation/lf-admin#667

## Type of change

- [ ] Bug fix
- [x] New feature
- [ ] Documentation update
- [ ] Refactoring / Tech debt
- [ ] CI/CD improvement
- [ ] Other:

## How to test

1. Run `npm run dev`.
2. Open `/about-us`.
3. Open the media gallery modal.
4. Search for a value that returns no results, for example `qwerty-no-image-test`.
5. Confirm that the empty state message is shown in the gallery area.

Also checked:

- `npx eslint app/shared/components/media-modal/views/gallery-view/GalleryView.tsx app/shared/components/media-modal/views/gallery-view/GalleryView.test.tsx app/shared/components/media-modal/views/shared-view.styles.ts`
- `npx prettier --check app/shared/components/media-modal/views/gallery-view/GalleryView.tsx app/shared/components/media-modal/views/gallery-view/GalleryView.test.tsx app/shared/components/media-modal/views/shared-view.styles.ts`
- `npm test -- --runTestsByPath app/shared/components/media-modal/views/gallery-view/GalleryView.test.tsx --runInBand --coverage=false`

## Video / Screenshots

image

## Checklist

- [x] Code compiles and runs correctly
- [x] Tests pass successfully
- [x] Linting checks are clean
- [x] No Sonar issues
- [ ] Documentation updated (if applicable)
- [x] All comments and requested changes addressed
- [x] Branch is up to date with the target branch
- [x] Linked issue/task included
- [x] Task moved to the review column on the board

### CODE REVIEW

**Comment by DenisGordProgrammer** [comment]
sonar issue here

**Comment by exity2905** [comment]
@DenisGordProgrammer I checked SonarCloud, and it shows 0 issues now. I only see a warning about file encoding, but no exact code issue. Could you please show me where the issue is if you still see it?

**Comment by vladashvch** [comment]
> @DenisGordProgrammer I checked SonarCloud, and it shows 0 issues now. I only see a warning about file encoding, but no exact code issue. Could you please show me where the issue is if you still see it?

Look at the 'Files changed' tab and open GalleryView.tsx. On the left side, you should see a triangular warning icon with an exclamation mark. If you scroll to the end of the file, you’ll see the issue there.

**Comment by exity2905** [comment]
@DenisGordProgrammer @vladashvch fixed, thanks. SonarCloud now shows 0 new issues.

File: app/shared/components/media-modal/views/gallery-view/GalleryView.tsx [hunk 1]
```diff
@@ -8,35 +8,53 @@ import { GalleryCard } from '../../components/gallery-card/GalleryCard';
...
     matchesFile: matchesAudio
   },
   pdf: {
     folder: 'uploads',
     assetType: AssetType.Pdf,
     title: 'Усі файли',
+    emptyStateText: {
+      uk: 'Файлів не знайдено. Спробуйте додати файли до медіатеки.',
+      en: 'No files found. Try adding files to the media library.'
+    },
```
Comment (vladashvch) [inline app/shared/components/media-modal/views/gallery-view/GalleryView.tsx]: This GalleryView component is used only in the admin interface and won't appear on the client side. Since the admin ui is only available in Ukrainian, it might be worth avoiding locale handling here and using Ukrainian only. Could you please recheck this?

Comment (exity2905) [reply app/shared/components/media-modal/views/gallery-view/GalleryView.tsx]: Thanks, you’re right. I removed locale handling here and kept Ukrainian-only empty state text for the admin UI.

**Review by DenisGordProgrammer — APPROVED** [review]
good

File: app/shared/components/media-modal/views/gallery-view/GalleryView.tsx [hunk 2]
```diff
@@ -20,23 +20,31 @@ type MediaKindConfig = {
...
   matchesFile?: (mimeType: string, filename: string) => boolean;
 };
 
 const MEDIA_KIND_CONFIG: Record<MediaKind, MediaKindConfig> = {
-  image: { folder: 'photos', assetType: AssetType.Image, title: 'Усі зображення' },
+  image: {
+    folder: 'photos',
+    assetType: AssetType.Image,
+    title: 'Усі зображення',
+    emptyStateText: 'Зображень не знайдено. Спробуйте додати зображення до медіатеки.'
```
Comment (dest411) [inline app/shared/components/media-modal/views/gallery-view/GalleryView.tsx:33]: The PR description states that `"The empty state uses localized Ukrainian and English text"`, but the strings in `MEDIA_KIND_CONFIG` are hardcoded in Ukrainian
also 39 and 47 lines

Comment (exity2905) [reply app/shared/components/media-modal/views/gallery-view/GalleryView.tsx:33]: Thanks, good catch. I updated the PR description to remove the outdated English localization mention.

**Review by dest411 — APPROVED** [review]

**Review by vladashvch — APPROVED** [review]
